docs/modules/ROOT/pages/tmail-backend/features.adoc

```diff
@@ -0,0 +1,52 @@
+= Advanced features
+:navtitle: Advanced features
+
+TMail offer number of essential advanced features, amongst them:
+
+image::lock.png[Lock logo]
+
+== PGP encryption
+
+
+TMail allow your users to store their emails encrypted in their mailbox.
+
+To do so, first:
+
+ - *Pre-requisite*: an administrator need to enable the encrypted mailbox. This can easily be done via the `mailbox.properties` configuration file.
```

Comment:
```suggestion
 - *Pre-requisite*: an administrator needs to enable the encrypted mailbox. This can easily be done via the `mailbox.properties` configuration file.
```